Libraries/PeriphDrivers/Include/MAX32672/adc.h

```diff
@@ -246,6 +246,7 @@ typedef struct {
     mxc_adc_avg_t avg_number; ///< no of samples to average
     mxc_adc_div_lpmode_t lpmode_divder; ///< Divide by 2 control in lpmode
     uint8_t num_slots; ///< num of slots in the sequence
+    int8_t dma_channel; ///< DMA channel number for DMA transfer
```

Comment:
We should consider updating this comment to say that users should call MXC_DMA_AcquireChannel to assign the value for this field.

Reply:
Updated that comment in the code.